website/docs/guides/dbt-python-bigframes.md

```diff
@@ -0,0 +1,384 @@
+
+```Some outputs of the Colab notebook from bigframes, which is stored in GCS```
+
+22:35:52  1 of 3 OK created python table model jialuo_test_us.my_bigframes_model ......... [OK in 228.85s]
+22:35:52  2 of 3 START sql table model jialuo_test_us.my_first_dbt_model ................. [RUN]
+22:35:55  2 of 3 OK created sql table model jialuo_test_us.my_first_dbt_model ............ [CREATE TABLE (2.0 rows, 0 processed) in 3.25s]
+22:35:55  3 of 3 START sql view model jialuo_test_us.my_second_dbt_model ................. [RUN]
+22:35:56  3 of 3 OK created sql view model jialuo_test_us.my_second_dbt_model ............ [CREATE VIEW (0 processed) in 0.96s]
+22:35:56  
+22:35:56  Finished running 2 table models, 1 view model in 0 hours 3 minutes and 55.30 seconds (235.30s).
+22:35:56  
+22:35:56  Completed successfully
+22:35:56  
+22:35:56  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
+```
+
+OR, just execute the selected model
+```
+> dbt run --select [your_bigframes_model]
+
+...
+```
+
+3. **You can observe the python execution in the dbt logs**
+For example
+```
+data:
+    text/html: ['Query job eaa77dfe-1e17-492e-815d-4728d65a07e3 is DONE. 0 Bytes processed. <a target="_blank" href="https://console.cloud.google.com/bigquery?project=bigframes-dev-perf&j=bq:US:eaa77dfe-1e17-492e-815d-4728d65a07e3&page=queryresults">Open Job</a>']
+    text/plain: ['<IPython.core.display.HTML object>']
+metadata:
+output_type:
+    display_data
+
+data:
+    text/html: ['Query job dfe0d09d-3579-4838-beb3-567f4f9543c0 is DONE. 0 Bytes processed. <a target="_blank" href="https://console.cloud.google.com/bigquery?project=bigframes-dev-perf&j=bq:US:dfe0d09d-3579-4838-beb3-567f4f9543c0&page=queryresults">Open Job</a>']
+    text/plain: ['<IPython.core.display.HTML object>']
+metadata:
+output_type:
+    display_data
+```
+
+4. *(Optional)* **You can also go to the Colab Enterprise Executions tab and GCS bucket from the GCP console to view the codes and logs (including previous executions).**
+
+The 'Runtime Templates' tab:
+
+<Lightbox src="/img/guides/gcp-guides/gcp-bigframes-runtime.png" title="Runtime Templates"/>
+
+Note:
+
+If no `default` runtime template is available, it will automatically create one for the user and mark it `default` for next time usage.
+To get the permissions that you need to create a runtime template in Colab Enterprise, ask your administrator to grant you the Colab Enterprise Admin (roles/aiplatform.colabEnterpriseAdmin) IAM role on the project ([runtime template link](https://cloud.google.com/colab/docs/create-runtime-template#:~:text=By%20creating%20a%20runtime%20template,characteristics%20based%20on%20your%20needs.)).
+
+The 'Executions' tab:
+<Lightbox src="/img/guides/gcp-guides/gcp-bigframes-executions.png" title="Executions"/>
+
+The 'Results' and 'Source Notebook' will also be written into the specified bucket:
+<Lightbox src="/img/guides/gcp-guides/gcp-bigframes-source.png" title="Source Notebook"/>
+
+## FAQs
```

Comment:
Some of this should just go into the BigQuery config page (which we can direct users to  at the beginning of the quickstart).  This way you aren't maintaining time specific context in a quickstart. 

A quickstart should just be a tutorial on how to get set up and use it, not a blog post about the feature  or the documentation site. Please remove this entire FAQ section from the quickstart.